http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/LeoPublisher.scala

```diff
@@ -118,6 +118,9 @@ final class LeoPublisher[F[_]](
           appQuery.updateStatus(m.appId, AppStatus.Stopping).transaction
         case m: LeoPubsubMessage.StartAppMessage =>
           appQuery.updateStatus(m.appId, AppStatus.Starting).transaction
+        case _: LeoPubsubMessage.UpdateAppMessage =>
+          F.unit
+          //TODO - See if we need to transition to an `UPDATING` status
```

Comment:
I would rather not do that as part of the MVP if we can avoid it

Reply:
Whats the logic behind this decision?

Reply:
Mostly consistency with the pre-existing update messages for disk and runtime

Reply:
Ah, so for google runtimes `Updating` is reserved for mirroring the actual google status of the runtime, which is only used for updating certain fields (we delete/re-create for a lot of 'updates' a user may request to their vm). In this case however, I think it is important to capture the concept that this leo resource is updating. Especially in a world where WSM is mirroring the cloud resource status already, it seems particularly relevant for leo to know "hey there is actually something going on here", especially since in this case we are doing the helm commands directly.

Reply:
Gotcha, I can do that, especially if we end up having some downtime, the status will be important 👍

Reply:
Yeah, also thinking in the context of `deleteAll`. Not sure if an app being updated is safe to be deleted

Reply:
Good point!

Reply:
This means we will also need to update
- the UI model for app statuses here: https://github.com/DataBiosphere/terra-ui/blob/dev/src/libs/ajax/leonardo/models/app-models.ts#L17
- And the corresponding display status mapping and type https://github.com/DataBiosphere/terra-ui/blob/dev/src/libs/ajax/leonardo/models/app-models.ts#L45

Reply:
Ahh, now I remember why we did not want to add this to the MVP, but it does feel safer to include it, even if it requires a small UI change

Reply:
Done :) https://github.com/DataBiosphere/terra-ui/pull/3983